app/v2/notifications/post_notifications.py

```diff
@@ -271,7 +271,7 @@ def process_letter_notification(*, letter_data, api_key, template, reply_to_text
 
     postcode = letter_data['personalisation']['postcode']
     if not is_a_real_uk_postcode(postcode):
-        raise ValidationError(message='A real UK postcode is required.')
+        raise ValidationError(message='postcode is not a real UK postcode`')
```

Comment:
Think we had a version that was ```Must be a real UK postcode```

Reply:
Should I change it to that then, it does sound clearer to me.

Reply:
If @quis and the team are happy with that.